### ISSUE TITLE
Extension doesn't work in Visual Studio 2012

### ISSUE THREAD

**Issue by FrostFlame64** [madskristensen/ImageOptimizer#9]
Tried installing the latest version of the Image Optimizer (3.0.24) in to my Visual Studio 2012 (Version 11.0.61030.00 Update 4).

After restarting, a message was displayed stating that the Package did not load correctly.
It stated more information could be found in the Activity Log.

Looking into it, the following entry was added to the Activity Log:

``` xml
  <entry>
    <record>523</record>
    <time>2015/05/05 13:12:16.814</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [ImageOptimizerPackage]</description>
    <guid>{BF95754F-93D3-42FF-BFE3-E05D23188B08}</guid>
    <hr>80004005 - E_FAIL</hr>
    <errorinfo>Could not load file or assembly 'Microsoft.VisualStudio.Shell.12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.</errorinfo>
  </entry>
```

PS: I've used your extension for years and loved it. Hopefully I be able to continue using it!

**Comment by stephendougherty** [comment]
Same error here Mads

``` <entry>
   <record>696</record>
    <time>2015/04/22 13:32:24.710</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [ImageOptimizerPackage]</description>
    <guid>{BF95754F-93D3-42FF-BFE3-E05D23188B08}</guid>
    <hr>80004005 - E_FAIL</hr>
    <errorinfo>Could not load file or assembly 'Microsoft.VisualStudio.Shell.12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.</errorinfo>
  </entry>
```

Thanks for your great work! Look forward to an update on this issue. 

Thanks
Stephen

**Comment by stephendougherty** [comment]
Thanks Mads!

On Wed, May 6, 2015 at 8:18 AM -0700, "Mads Kristensen" notifications@github.com wrote:
Closed #9 via 4bdaa9ccc50c8dc0860d44ea416425e7dd1b7ba5.

---

Reply to this email directly or view it on GitHub:
https://github.com/madskristensen/ImageOptimizer/issues/9#event-298334589
